course/api/full_serializers.py

```diff
@@ -219,3 +223,16 @@ class Meta(CourseStudentGroupBriefSerializer.Meta):
             }
         }
 
+
+class TreeCourseModuleSerializer(serializers.Serializer):
+    id = serializers.IntegerField()
```

Comment:
I wonder if we should include other additional `CourseModule` fields in this serializer.

Reply:
The IntelliJ team was happy with the fields currently available in the response. But if you think some specific fields should be added, I can do that.